Sources/Helpers/OAUtilities.m

```diff
@@ -2179,4 +2180,45 @@ + (NSString *) getFormattedValue:(NSString *)value unit:(NSString *)unit separat
     return [NSString stringWithFormat:separateWithSpace ? OALocalizedString(@"ltr_or_rtl_combine_via_space") : @"%@%@", value, unit];
 }
 
++ (NSString *) buildGeoUrl:(NSString *)latitude longitude:(NSString *)longitude zoom:(int)zoom
+{
+    return [NSString stringWithFormat:@"geo:%@,%@?z=%i", latitude, longitude, zoom];
+}
+
++ (void)copyToClipboardWithToast:(NSString *)text inView:(UIView *)view
+
+{
+    UIPasteboard *pb = [UIPasteboard generalPasteboard];
+    [pb setString:text];
+
+    [self showToast:OALocalizedString(@"copied_to_clipboard") details:text duration:4 inView:view];
```

Comment:
Please do not add this functionality to this method. Ideally, a utility method should perform only one action (in this case copy to clipboard)